### DISCUSSION TITLE
I wish to add company_id column

### DISCUSSION THREAD

**Discussion by ShahanurSharif** [spatie/laravel-activitylog#815] [Q&A] [answered]
Hello I am developing multi-tanent software. I have used company_id everywhere. There is anyway to add company_id column though I did not understand tap method clearly. 

Detail answer will be greatly appreciated.

**Comment by Gummibeer** [comment, marked as answer]
Hey,
the easiest way to do this would be an observer for the activity model.
Another solution would be to override and rebind the `` class and adjust one of the methods:
https://github.com/spatie/laravel-activitylog/blob/ade270f291f4cb5883b3653919304a0b4e1cc284/src/ActivityLogger.php#L63-L74
https://github.com/spatie/laravel-activitylog/blob/ade270f291f4cb5883b3653919304a0b4e1cc284/src/ActivityLogger.php#L225-L235

The observer is the easiest way as it will not conflict with future releases and you don't have to copy code from the package.
